src/content/quickstart/verify_with_discovery.md

```diff
@@ -16,9 +16,25 @@ kubectl -n default  run --generator=run-pod/v1 tmp-shell --rm -i --tty --image q
 curl nginx.default.svc.supercluster.local:8080
 ```
 
+To verify a headless service follow the steps below using a mehdb service
+
+```bash
+export KUBECONFIG=cluster-b/auth/kubeconfig
+kubectl apply -f https://raw.githubusercontent.com/openshift-evangelists/mehdb/master/app.yaml
+subctl export service --namespace default mehdb
+```
+
+```bash
+export KUBECONFIG=cluster-a/auth/kubeconfig
```

Comment:
This seems superfluous, esp when appearing multiple times in same doc. This implies users need to set env variable everytime.

Reply:
but this helps in copy pasting right? should we make it an argument of the command?

Reply:
If user is expected to run these in a sequence, having it in first command of sequence should be enough.

Reply:
Done.